src/screens/TopicMemberScreen/index.js

```diff
@@ -54,9 +61,13 @@ const TopicMemberScreen = () => {
 
   const opacityAnimationHeader = React.useRef(new Animated.Value(1)).current;
 
+  const coverPath = topicDetail?.cover_path || null;
+
   const animatedHeight = React.useRef(
     new Animated.Value(
-      dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT + dimen.size.TOPIC_FEED_HEADER_HEIGHT
+      (coverPath
+        ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+        : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) + dimen.size.TOPIC_FEED_HEADER_HEIGHT
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `animatedHeight` variable has been updated to accommodate the presence of a cover image. The logic seems correct as it checks if `coverPath` exists and then uses the appropriate height value.
```diff
- const animatedHeight = React.useRef(
-     new Animated.Value(
-      dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT + dimen.size.TOPIC_FEED_HEADER_HEIGHT
-     )
-   ).current;
+ const animatedHeight = React.useRef(
+    new Animated.Value(
+       (coverPath
+         ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT_COVER
+         : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT) + dimen.size.TOPIC_FEED_HEADER_HEIGHT
+     )
+   ).current;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->